try {
    process.nextTick(function () {
        throw new Error();
    });
} catch (err) {
    console.log('Exception caught:', err);
}

//# sourceMappingURL=fail-compiled.js.map

//# sourceMappingURL=fail-compiled-compiled.js.map